packages/react/src/runtimes/edge/createEdgeRuntimeAPI.ts

```
@@ -115,7 +120,29 @@ export const getEdgeRuntimeStream = async ({
+  const [t1, t2] = stream.tee();
+  stream = t1;
+
+  t2.pipeTo(
+    new WritableStream({
+      write(chunk) {
+        console.log(chunk);
+      },
+    }),
+  );
+
+  const [t1, t2] = stream.tee();
```

Biome (1.9.4)

[error] Shouldn't redeclare 't1'. Consider to delete it or rename it.

't1' is defined here:


(lint/suspicious/noRedeclare)